easybuild/easyconfigs/f/Ferret/Ferret-7.6.0-foss-2022a.eb

```diff
@@ -0,0 +1,53 @@
+name = 'Ferret'
+version = '7.6.0'
+homepage = 'https://ferret.pmel.noaa.gov/Ferret/'
+description = """Ferret is an interactive computer visualization and analysis environment
+designed to meet the needs of oceanographers and meteorologists analyzing large and complex gridded data sets."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+toolchainopts = {'usempi': True, 'pic': True, 'extra_fflags': '-fallow-argument-mismatch -fallow-invalid-boz'}
+
+source_urls = ['https://github.com/NOAA-PMEL/Ferret/archive/']
+sources = ['v%s.tar.gz' % version]
+checksums = ['69832d740bd44c9eadd198a5de4d96c4c01ae90ae28c2c3414c1bb9f43e475d1']
+
+builddependencies = [
+    ('Autotools', '20220317'),
+]
+
+dependencies = [
+    ('X11', '20220504'),
+    ('netCDF-Fortran', '4.6.0'),
+    ('zlib', '1.2.12'),
+    ('Szip', '2.1.1'),
+    ('cURL', '7.83.0'),
+    ('ncurses', '6.3'),
+    ('libreadline', '8.1.2'),
+    ('tcsh', '6.24.01'),
+    ('Java', '11', '', SYSTEM),
+]
+
+# /bin/csh is required for installation by bin/make_executable_tar, bin/make_environment_tar
+# and also required by many installed scripts in bin/
+osdependencies = ['tcsh']
```

Comment:
`tcsh` is in the dependencies. Is this needed as an OS dep as well?

Reply:
ah yeah, this is indeed redundant, fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/19415/commits/360c1c3949fff5fec2298c3e8ba3f87d9ff0760f thanks